astro/src/content/blog/authcon-press-release.mdx

```diff
@@ -0,0 +1,51 @@
+---
+publish_date: 2024-12-03 
+title: FusionAuth Announces First Customer Identity and Access Management (CIAM) Conference
+description: AuthCon empowers product and engineering teams with practical solutions for modern CIAM needs
+authors: Story Tweedie-Yates
+image: /img/blogs/authcon-press-release/authcon-press-release.png
+tags: press release, CIAM, AuthCon
+categories: Company
+excerpt_separator: "{/* more */}"
+---
+
+FusionAuth Announces First Customer Identity and Access Management (CIAM) Conference
```

Comment:
```suggestion
```

This is duplicated because it is already the title.

Reply:
I don't know enough yet to know what might be duplicated or not - will trust your know-how to make sure nothing is duplicated in the page in terms of the header and subhead. Thanks!